app-template/copay/sass/app.scss

```diff
@@ -0,0 +1,754 @@
+/***********************************************************************************************************
+ *
+ * Variables
+ * 
+ ***********************************************************************************************************/
+
+/*
+ * Define color palettes
+ */
+
+$color-palettes: (
```

Comment:
This creates a somewhat-leaky abstraction of our actual variables:

https://github.com/bitpay/copay/blob/master/src/sass/variables.scss

We're using Ionic v1's framework of variables, and only overriding things where truly necessary. (Otherwise our CSS will expand until it's unmaintainable, it will eventually become "write-only", and we'll ultimately have to trash it and start over.)